crates/biome_html_parser/src/syntax/mod.rs

```diff
@@ -74,7 +74,7 @@ fn parse_element(p: &mut HtmlParser) -> ParsedSyntax {
     }
     let m = p.start();
 
-    p.bump(T![<]);
+    p.expect(T![<]);
```

Comment:
We usually don't use `expect` after a check, because it's not needed. This method raises a diagnostic. That's why we use bump.